scripts/start.sh

```diff
@@ -2,4 +2,4 @@
 ethermintd --home /ethermint/node$ID/ethermintd/ start > ethermintd.log &
 sleep 5
 ethermintcli rest-server --laddr "tcp://localhost:8545" --chain-id 7305661614933169792 --trace > ethermintcli.log &
-tail -f /dev/null
\ No newline at end of file
+tail -f ethermintd.log
```

Comment:
This should be `/dev/null`. this is a workaround way to keep the script running and redirect output. user can tail manually to view the output.

Reply:
OK, the command line``docker logs -f ethermintdnode0
 in the log view in the [document ](https://docs.ethermint.zone/quickstart/testnet.html)can be removed.

Reply:
I'll change it again. https://github.com/ChainSafe/ethermint/pull/447